### PR TITLE
Documentation for coordinator

### PR DESCRIPTION
**Pull request by mikaelforeseeti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -402,6 +402,87 @@ model.disable_attackstep("S3Bucket", "ReadObject")
...
+# get token
+client = enterprise.client(
+    base_url=backend_base_url, username=USERNAME, password=PASSWORD
+)
+token = client._get_access_token()  # pylint: disable=protected-access
+header = {"Authorization": f"JWT {token}"}
+
+# post request
+resp = requests.post(f"{coordinator_base}/jobs", json=test_data, headers=header)
+resp.raise_for_status()
```
Comment (eringdahl) [inline README.md:469]: I think we can remove this part for now

File: README.md [hunk 2]
```diff
@@ -402,6 +402,87 @@ model.disable_attackstep("S3Bucket", "ReadObject")
...
+                        "object_name": "i-1",
+                        "attackstep": "HighPrivilegeAccess",
+                    },
+                },
+            ],
+        },
+    ],
+}
+
+host = "https://enterpriseinstance.foo"
```
Comment (eringdahl) [inline README.md:456]: Let's assume that they have a `client` already from example.py

File: README.md [hunk 3]
```diff
@@ -402,6 +402,87 @@ model.disable_attackstep("S3Bucket", "ReadObject")
...
+## Batch scenario operations
+
+Enterprise also supports batch scenario operations, though currently not through the SDK.
+
+You can POST json data to `HOST/coordinator/v1/jobs`. This endpoint requires a valid JWT token from the corresponding Enterprise instance.
+
+Here's a brief example of input data, using output json data from the securicad-aws-collector.
+
+```python
+
```
Comment (eringdahl) [inline README.md:414]: Can we assume that they have the `aws_data` from the example.py?

File: README.md [hunk 4]
```diff
@@ -402,6 +402,87 @@ model.disable_attackstep("S3Bucket", "ReadObject")
 model_info.save(model)
 ```
 
+## Batch scenario operations
+
+Enterprise also supports batch scenario operations, though currently not through the SDK.
```
Comment (eringdahl) [inline README.md:407]: Suggestion:
Enterprise also supports batch scenario operations through the API

File: README.md [hunk 5]
```diff
@@ -180,7 +180,7 @@ If you have installed securiCAD Enterprise with a client certificate or have rec
 ### Managed instance
 To use your `.p12` certificate file with the SDK, you need to extract the `.crt` and `.key` files. Run the following snippets to extract the required files. When prompted for the Import Password, use the one provided to you by foreseeti.
 
-`openssl pkcs12 -in cert.p12 -nocerts -out cert.key -nodes`
+`openssl pkcs12 -in cert.p12 -nocerts -out cert.key -nodes`\
```
Comment (maxwalls) [inline README.md:183]: Delete `\` at end of line

File: README.md [hunk 6]
```diff
@@ -402,6 +402,87 @@ model.disable_attackstep("S3Bucket", "ReadObject")
...
+
+# get token
+client = enterprise.client(
+    base_url=backend_base_url, username=USERNAME, password=PASSWORD
+)
+token = client._get_access_token()  # pylint: disable=protected-access
+header = {"Authorization": f"JWT {token}"}
+
+# post request
+resp = requests.post(f"{coordinator_base}/jobs", json=test_data, headers=header)
```
Comment (maxwalls) [inline README.md:468]: You could re-use the session in the client:
```python
client._session.post(f"{coordinator_base}/jobs", json=test_data)
```

File: README.md (hunk 5 above)
Comment (mikaelforeseeti) [reply README.md:183]: That was to force a linebreak

**Review by eringdahl — APPROVED** [review]

**Pull request merged**